receiver/prometheusremotewritereceiver/receiver_test.go

```diff
@@ -219,35 +234,149 @@ func TestTranslateV2(t *testing.T) {
-				mAttributes2.PutStr("d", "e")
-				mAttributes2.PutStr("foo", "bar")
-
+				parseJobAndInstance(rm2.Resource().Attributes(), "foo", "bar")
+				sm2 := rm2.ScopeMetrics().AppendEmpty()
+				sm2.Scope().SetName(defaultBuildName)
+				sm2.Scope().SetVersion(defaultBuildVersion)
+				m3 := sm2.Metrics().AppendEmpty().SetEmptyGauge()
+				dp3 := m3.DataPoints().AppendEmpty()
+				dp3.Attributes().PutStr("d", "e")
+				dp3.Attributes().PutStr("foo", "bar")
+				return expected
+			}(),
+			expectedStats: remote.WriteResponseStats{},
+		},
+		{
+			name: "provided otel_scope_name and otel_scope_version",
+			request: &writev2.Request{
+				Symbols: []string{
+					"", "__name__", "metric_with_scope",
+					"otel_scope_name", "custom_scope",
+					"otel_scope_version", "v1.0",
+					"job", "service-y/custom",
+					"instance", "instance-1",
+				},
+				Timeseries: []writev2.TimeSeries{
+					{
+						Metadata:   writev2.Metadata{Type: writev2.Metadata_METRIC_TYPE_GAUGE},
+						LabelsRefs: []uint32{1, 2, 3, 4, 5, 6, 7, 8, 9, 10},
+						Samples:    []writev2.Sample{{Value: 10, Timestamp: 100}},
+					},
+				},
+			},
+			expectedMetrics: func() pmetric.Metrics {
+				expected := pmetric.NewMetrics()
+				rm := expected.ResourceMetrics().AppendEmpty()
+				parseJobAndInstance(rm.Resource().Attributes(), "service-y/custom", "instance-1")
+				sm := rm.ScopeMetrics().AppendEmpty()
+				sm.Scope().SetName("custom_scope")
+				sm.Scope().SetVersion("v1.0")
+				_ = sm.Metrics().AppendEmpty().SetEmptyGauge().DataPoints().AppendEmpty()
+				return expected
+			}(),
+			expectedStats: remote.WriteResponseStats{},
+		},
+		{
+			name: "missing otel_scope_name/version falls back to BuildInfo",
+			// When missing, ls.Get returns "" so the defaults from BuildInfo are preserved.
+			request: &writev2.Request{
+				Symbols: []string{
+					"",                // index 0
+					"__name__",        // index 1
+					"metric_no_scope", // index 2
+					"job",             // index 3
+					"service-z/xyz",   // index 4
+					"instance",        // index 5
+					"inst-42",         // index 6
+					"d",               // index 7
+					"e",               // index 8
+					"foo",             // index 9
+					"bar",             // index 10
+				},
+				Timeseries: []writev2.TimeSeries{
+					{
+						Metadata:   writev2.Metadata{Type: writev2.Metadata_METRIC_TYPE_GAUGE},
+						LabelsRefs: []uint32{1, 2, 3, 4, 5, 6, 7, 8, 9, 10},
+						Samples:    []writev2.Sample{{Value: 5, Timestamp: 50}},
+					},
+				},
+			},
+			expectedMetrics: func() pmetric.Metrics {
+				expected := pmetric.NewMetrics()
+				rm := expected.ResourceMetrics().AppendEmpty()
+				parseJobAndInstance(rm.Resource().Attributes(), "service-z/xyz", "inst-42")
+				sm := rm.ScopeMetrics().AppendEmpty()
+				// Expect fallback to default BuildInfo.
+				sm.Scope().SetName(defaultBuildName)
+				sm.Scope().SetVersion(defaultBuildVersion)
```

Comment:
Maybe set the name and version as empty strings to make explicit the value that will be assigned for them?
```suggestion
				sm.Scope().SetName("")
				sm.Scope().SetVersion("")
```